src/System.Net.Http/tests/FunctionalTests/HttpClientHandlerTest.cs

```diff
@@ -702,7 +703,62 @@ public static IEnumerable<object[]> VerifyUploadServersStreamsAndExpectedData
                 }
             }        
         }
-        
+        #endregion
+
+        #region Version tests
+        [ActiveIssue(4754, PlatformID.Windows)]
+        [Theory]
+        [MemberData("Http2Servers")]
+        public async Task SendAsync_RequestVersion10_ResponseVersion10(Uri server)
```

Comment:
nit: Why is the MemberData() on its own line?  All the other Theory tests have it as part of the Theory attribute.